bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/AbstractMyStromHandler.java

```diff
@@ -74,13 +63,11 @@ public abstract class AbstractMyStromHandler extends BaseThingHandler {
     public AbstractMyStromHandler(Thing thing, HttpClient httpClient) {
         super(thing);
         this.httpClient = httpClient;
+        config = getConfigAs(MyStromConfiguration.class);
```

Comment:
This should be done within `initialize()`. Otherwise configuration will not be refreshed after being modified.